### DISCUSSION TITLE
Add new submitters to a submission

### DISCUSSION THREAD

**Discussion by fersilva16** [docusealco/docuseal#240] [Q&A] [answered]
Is there a way to add new submitters or edit the submission? I need signatures in a dynamic form where I don't have all the submitters at first. I took a look at the API and I didn't find anything related, let me know if I'm missing something. Thank you!

**Comment by omohokcoj** [comment, marked as answer]
@fersilva16 adding/removing submitters from an already created submission is not possible - but you can add dummy submitters and then update their emails and send signature requets via the PUT /submitter/id API

**Reply by fersilva16** [reply]
Thank you @omohokcoj, I will follow with this solution!


There are plans for adding this feature in the future?
